pkg/nsxt/nsxt_connector.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/vmware/vsphere-automation-sdk-go/runtime/protocol/client"
 	"github.com/vmware/vsphere-automation-sdk-go/runtime/security"
 	"k8s.io/cloud-provider-vsphere/pkg/nsxt/config"
+	"k8s.io/klog"
```

Comment:
should we use `k8s.io/klog/v2` instead?

Reply:
I see other files are using `k8s.io/klog`, do we plan to use v2 instead of v1?

Reply:
klog v2 was just merged. We should avoid mixing between the two. Mixing versions would also require a different init for both libraries. Easier just to avoid it.

Reply:
+1 to use klog/v2

Reply:
Got it, thanks